docs/usage.rst

```diff
@@ -1,130 +0,0 @@
-these files are over-writeable, but they should only be touched if one knows very well
-what they are doing.
-
-Finally, ``21cmFAST`` contains a more robust cataloguing/caching method. Instead of
-saving data with a selection of the dependent parameters written into the filename --
-a method which is prone to error if a parameter which is not part of that selection is
-modified -- ``21CMMC`` writes all data into a configurable central directory with a hash
-filename unique to *all* parameters upon which the data depends. Each kind of dataset has
-attached methods which efficiently search this central directory for matching data to be
-read in when necessary.
-Several arguments are available for all library functions which produce such datasets
-that control this output. In this way, the data that is being retrieved is always
-reliably produced with the desired parameters, and users need not concern themselves
-with how and where the data is saved -- it can be retrieved merely by creating an empty
-object with the desired parameters and calling ``.read()``, or even better, by calling
-the function to *produce* the given dataset, which will by default just read it in if
-available.
-
-It should not be of great concern to the user, but in the case that it is, the datasets
-themselves are stored as HDF5 files, and the various parameters defining them are stored
-as attributes.
-
-CLI
-===
-The CLI interface always starts with the command ``21cmfast``, and has a number of
-subcommands. To list the available subcommands, use::
-
-    $ 21cmfast --help
-
-To get help on any subcommand, simply use::
-
-    $ 21cmfast <subcommand> --help
-
-Any subcommand which runs some aspect of ``21cmFAST`` will have a ``--config`` option,
-which specifies a configuration file (by default this is
-``~/.21cmfast/runconfig_example.yml``). This config file specifies the parameters of the
-run. Furthermore, any particular parameter that can be specified in the config file can
-be alternatively specified on the command line by appending the command with the
-parameter name, eg.::
-
-    $ 21cmfast init --config=my_config.yml --HII_DIM=40 hlittle=0.7 --DIM 100 SIGMA_8 0.9
-
-The above command shows the numerous ways in which these parameters can be specified
-(with or without leading dashes, and with or without "=").
-
-The CLI interface, while simple to use, does have the limitation that more complex
-arguments than can be passed to the library functions are disallowed. For example,
-one cannot pass a previously calculated initial conditions box to the ``perturb``
-command. However, if such a box has been calculated with the default option to write it
-to disk, then it will automatically be found and used in such a situation, i.e. the
-following will not re-calculate the init box::
-
-    $ 21cmfast init
-    $ 21cmfast perturb redshift=8.0
-
-This means that almost all of the functionality provided in the library is accessible
-via the CLI.
-
-
-Library
```

Comment:
I removed the usage section about the library because there are numerous tutorials and also full API documentation, which makes it redundant and in danger of growing out of date.